Formula/pulp.rb

```diff
@@ -7,6 +7,11 @@ class Pulp < Formula
   sha256 "77b8a5ff21291257766f54289dbe92d2558fc64e9a034513647a76400c0e7d94"
   license "LGPL-3.0"
 
+  livecheck do
+    url :url
```

Comment:
```suggestion
    url :stable
```
While `url :url` works (as it's supported/used in Casks), perhaps we want `:stable` here?

Reply:
Yes, thanks for the catch! I've been working on a bunch of `livecheck` blocks in homebrew/cask recently and it clearly shows here, ahah.

I've been reworking the `livecheck` block Rubocops to also apply to casks (though I'm fixing the issues in the cask taps before creating a brew PR) and I'm working on some tweaks that should catch this particular situation in the future. This definitely isn't the first time I've made this silly mistake due to switching back and forth between homebrew/core and homebrew/cask.